nucypher/blockchain/eth/agents.py

```diff
@@ -118,15 +118,15 @@ def approve_transfer(self, amount: int, target_address: str, sender_address: str
         """Approve the transfer of token from the sender address to the target address."""
         payload = {'gas': 500_000}  # TODO #413: gas needed for use with geth.
         contract_function = self.contract.functions.approve(target_address, amount)
-        receipt = self.blockchain.send_transaction(transaction_function=contract_function,
+        receipt = self.blockchain.send_transaction(contract_function=contract_function,
```

Comment:
Yeah, this is much nicer. :+1:

Reply:
Yeah - keeping with the conventional name here.